tests/core/test_polydata.py

```diff
@@ -1234,7 +1234,7 @@ def test_empty_regular_faces():
 
 
 def test_regular_faces_mutable():
-    points = [[1, 1, 1], [-1, 1, -1], [1, -1, -1], [-1, -1, 1]]
+    points = [[1.0, 1, 1], [-1, 1, -1], [1, -1, -1], [-1, -1, 1]]
```

Comment:
If this is not what intended, it is better to revert.
```suggestion
    points = [[1, 1, 1], [-1, 1, -1], [1, -1, -1], [-1, -1, 1]]
```

Reply:
This is intended, that's why it's there. This causes the resulting numpy array to be float rather than int, suppressing another warning.

This is admittedly lazy, but there was already precedent for this in the module and I wanted to make minimal changes to resolve trivial warmings.

If you'd like I can make these hacky cases nicer, at the cost of a larger unrelated diff.

Reply:
Thank you. Can we add our intent in the comments?

Reply:
It felt too lazy to slap a comment on what's really just a lazy kludge, so I decided to clean these up properly in https://github.com/pyvista/pyvista/pull/6156/commits/bf0c73eb8237c63d7d9c067addea8236210b049d.